ste/folderCreationTracker.go

```diff
@@ -167,17 +243,22 @@ func (f *jpptFolderTracker) StopTracking(folder string) {
 		return // Not possible to track this
 	}
 
+	//add a log mentioning we are in stotracking and folder is being deleted
```

Comment:
Typo stotracking

Reply:
Added for debugging purposes. Removed now.